lib/inferno/apps/web/controllers/test_runs/create.rb

```diff
@@ -10,7 +10,7 @@ class Create < Controller
           def call(params)
             test_run = repo.create(create_params(params))
             inputs = (params[:inputs] || {}).each_with_object({}) do |input, new_inputs|
-              new_inputs[input[:name].to_sym] = input[:value]
+              new_inputs[input[:key].to_sym] = input[:value]
```

Comment:
Why the changes from `name` to `key`? `name` seems more consistent with other bits of inferno (like requests).

Reply:
I liked key better because I thought name and title were too similar, but I changed it back to name to be more consistent with the rest of inferno.